### PR TITLE
Use debug parameter to make debug script optional

### PR DESCRIPTION
**Pull request by TimboTambo**
# Make debug parameter optional for GA script

Makes debug script optional.
Relates to issue #446

### Acceptance Criteria

- [ ] Setting debug parameter is optional and controlled by environment variable

### Testing information

See issue

### Pre-flight Checklist

- [ ] Testing aligns with [testing strategy][testing strategy]
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ stages:
               export AZURE_SEARCH_WORD_FUZZINESS=1
               export GOOGLE_GTM_CONTAINER_ID=GTM-WJ5TW34
               export GOOGLE_TRACKING_ID=UA-6838115-11
+              GOOGLE_USE_DEBUG=true
```
Comment (craiga) [inline azure-pipelines.yml]: Should have `export `?

Comment (craiga) [reply azure-pipelines.yml]: Also, do we want debug to be on in production?

Comment (TimboTambo) [reply azure-pipelines.yml]: Yes, yes it should have `export` - good spot. This pipeline is release to dev environment when merged to master.

**Comment by m-doughty** [comment]
QA passed.

**Pull request merged**